homeassistant/components/tibber/__init__.py

```diff
@@ -53,6 +52,11 @@ async def _close(event: Event) -> None:
 
     try:
         await tibber_connection.update_info()
+        if tibber_connection.name == "":
```

Comment:
Maybe just check for falsy?
```suggestion
        if not tibber_connection.name:
```

Reply:
Why doesn't the library raise an exception if there was a failure? It would be more clear to do that so we can catch that and re-raise to retry as appropriate.

Reply:
Yes, I think we should change pyTibber to raise an exception if it is a failure.

Reply:
@Danielhiversen do you want to wait for the library change or should we merge here meanwhile?

Reply:
I guess it is ok for now.

Reply:
I actually started that work too but I believe this will help a lot to ease the pressure from the users in the meanwhile